### PR TITLE
add a new doc: Meetup Best Practices

### PR DESCRIPTION
**Pull request by Smana**

### CODE REVIEW

File: meetups/Meetup_Best_Practices.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+(e.g. Cloud Computing · DevOps · Native Cloud Computing · Docking · Microservices · Kubernetes · Cloud Native · Delivery ...)
+
+The most important thing is to keep a **active group**, with frequent events (ideally once a month). <br>
+Animating a group alone can be difficult to manage, if it is possible you would be several co-organizers.
+
+
+* **Find Speakers**
+
+You should provided some kind of **form** in order to allow people to submit their talk proposal as follows :
+https://docs.google.com/forms/d/1xO78iKdMEzvs11E1gmDBVYeJU-Q6h4GG-IJbyA1YzLk/viewform <br>
```
Comment (idvoretskyi) [inline meetups/Meetup_Best_Practices.md]: @Smana looks good. Can you create a generic version of this (Meetup Group/location neutral)?

Comment (Smana) [reply meetups/Meetup_Best_Practices.md]: yes, i'm doing that :+1:

File: meetups/Meetup_Best_Practices.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+## Meetups Best Practices
+
+This page allows you to start a new meetup group based on the experience of existing ones. <br>
+Here is a list of suggestions and tips to get you started:
+
+* **Create the meetup**
+
```
Comment (Smana) [inline meetups/Meetup_Best_Practices.md]: Maybe i should add a word on the naming of the meetup group ? 
`Cloud Native Computing <Location>` given the current meetups list.
What do you think @idvoretskyi ?

Comment (idvoretskyi) [reply meetups/Meetup_Best_Practices.md]: @Smana we also accept Meetup groups with the different names, reflecting specific CNCF project (eg. Kubernetes Meetup Group or Prometheus Meetup Group).

Comment (Smana) [reply meetups/Meetup_Best_Practices.md]: right, forget it then :)

File: meetups/Meetup_Best_Practices.md (hunk 1 above)
Comment (Smana) [reply meetups/Meetup_Best_Practices.md]: i shared with you a form, waiting for your comments

Comment (idvoretskyi) [reply meetups/Meetup_Best_Practices.md]: @Smana perfect, added a few comments.

File: meetups/Meetup_Best_Practices.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Animating a group alone can be difficult to manage, if it is possible you would be several co-organizers.
+
+
+* **Find Speakers**
+
+You should provided some kind of **form** in order to allow people to submit their talk proposal as follows :
+
+https://docs.google.com/forms/d/1V2Y03YMOrIor0M796_WMbYx-fdsn80ngaT-PIum8gUU/edit <br>
+_**Note**_: Do a copy ! Please do not edit the form in place.
+
```
Comment (Smana) [inline meetups/Meetup_Best_Practices.md]: @idvoretskyi i've applied the changes according to your suggestions

Comment (idvoretskyi) [reply meetups/Meetup_Best_Practices.md]: LGTM

File: meetups/Meetup_Best_Practices.md (hunk 2 above)
Comment (idvoretskyi) [reply meetups/Meetup_Best_Practices.md]: Feel free to add this as a recommendation.

Comment (Smana) [reply meetups/Meetup_Best_Practices.md]: i added a recommendation, please tell me if that's ok for you @idvoretskyi

File: meetups/Meetup_Best_Practices.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+Animating a group alone can be difficult to manage, if that's possible you would be a team composed of passionate co-organizers.
+
+
+* **Find Speakers**
+
+You should provided some kind of **form** in order to allow people to submit their talk proposal as follows :
+
+https://docs.google.com/forms/d/1V2Y03YMOrIor0M796_WMbYx-fdsn80ngaT-PIum8gUU/edit <br>
+_**Note**_: Do a copy ! Please do not edit the form in place.
+
```
Comment (rbenzair) [inline meetups/Meetup_Best_Practices.md]: hi @Smana could you ad my suggestion bellow 
could you ad my suggestion below 

**Booking** 
If you have limit room on place or capacity 

- You should do an overbooking, there are around 30% of people book and they never come. for example if ou have only 50 places, you can at least open for 60 places.

- You must think to send again a reminder message in the meetup group to ask the people to free their place if he did not comes.

**Comment by Smana** [comment]
@idvoretskyi i've added changes following @rbenzair and @leecalcote suggestions.
I've put them in a distinct commit but if they're ok for you i will squash it.

**Comment by idvoretskyi** [comment]
@Smana 

> I've put them in a distinct commit but if they're ok for you i will squash it.

Looks perfect, please, squash, and I'm ready to merge.

**Comment by Smana** [comment]
@idvoretskyi done :)

**Comment by Smana** [comment]
@idvoretskyi , is there anything missing ?

**Comment by idvoretskyi** [comment]
@Smana perfect, thank you!

**Comment by idvoretskyi** [comment]
@Smana perfect, thank you!

**Pull request merged**